### DISCUSSION TITLE
[Feature] possibilities for partners

### DISCUSSION THREAD

**Discussion by juanpeich** [immich-app/immich#11147] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, my wife and I have 2 accounts within Inmich, each with its own username. We have shared the albums that interest us and we both see each other's photos on the timeline. Could the following functions be done? 

1- Choose visibility in the partner's timeline for each of the albums. This way, if I have an album created on another topic that does not interest even a woman, the photos corresponding to that album will not be visible on her timeline. 

2- Possibility of giving full access to the partner. In this way, the partner could delete duplicate or incorrect photos on their timeline, even if they belong to the other user. 

3-  Share facial recognition. we share children, grandparents, etc. They are the same faces for both. If they could be merged, it would be ideal (even if it is as an option in the user settings)

4- Possibility of adding a folder with music to make memories. 

5- This is a question: could I pass photos through samba, within immich's folders and he would recognize them, or do I have to use the upload option? 

Thank you very much for all your work. really immich is spectacular

### Platform

- [x] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
2: More advanced sharing is in the works, and will eventually support this.
5: You want external libraries, see the docs

Most of these are duplicates of existing feature requests.

**Reply by juanpeich** [reply]
Thank yo soy much for answering. 

I'm not referring to an external library but rather putting it in it. 
I already have a hierarchy: year - month - *.jpg 

I mean if it is possible to put the folders directly through Samba into the already created library. 

Greetings and thank you very much for your help.
